projects/plugins/jetpack/modules/subscriptions/subscribe-overlay/subscribe-overlay.css

```diff
@@ -0,0 +1,83 @@
+body.jetpack-subscribe-overlay-open {
+	overflow: hidden;
+}
+
+.jetpack-subscribe-overlay {
+	--jetpack-subscribe-overlay--text-color: var(--wp--preset--color--primary, black);
+	--jetpack-subscribe-overlay--background-color: var(--wp--preset--color--background, var(--wp--preset--color--base, var(--wp--preset--color--contrast, #F9F9F9)));
+
+	visibility: hidden;
+	position: fixed;
+	z-index: 50000; /* Same as WP.com Action bar */
+	left: 0;
+	top: 0;
+	width: 100%;
+	height: 100%;
+	overflow: hidden;
+	background-color: transparent;
+	transition: all 0.4s;
+	color: var(--jetpack-subscribe-overlay--text-color);
+}
+
+.jetpack-subscribe-overlay__content {
+	position: relative;
+	visibility: hidden;
+	overflow: hidden;
+	top: 100%;
+	margin: 15% auto;
+	width: 100%;
+	max-width: 600px;
+	transition: all 0.4s;
+	text-wrap: pretty;
+}
+
+.jetpack-subscribe-overlay__close {
+	display: none;
+	cursor: pointer;
+	position: absolute;
+	top: 32px;
+	right: 32px;
+	width: 24px;
+	height: 24px;
+}
+
+body.admin-bar .jetpack-subscribe-overlay__close {
+	top: 64px;
+}
+
+.jetpack-subscribe-overlay__to-content {
+	display: none;
+	position: fixed;
+	bottom: 64px;
+	left: 0;
+	right: 0;
+	margin: 0 auto;
+
+	a {
+		color: var(--jetpack-subscribe-overlay--text-color);
+	}
+}
+
+.jetpack-subscribe-overlay.open {
+	background-color: var(--jetpack-subscribe-overlay--background-color);
```

Comment:
Same as above about styling https://github.com/Automattic/jetpack/pull/37372/files#r1599865348

Reply:
I need to set the background color, since without it the overlay will be transparent.

The issue I don't know how to solve is allowing users to change it.

Currently, if you change the background color it looks like this:

<img width="1083" alt="Screenshot 2024-05-16 at 10 32 22" src="https://github.com/Automattic/jetpack/assets/4068554/fa07ce11-2324-4cef-ac21-6062ad712adc">

The fix for it would be to wrap the content with another group block and stretch it to the entire screen size, WDYT?

Reply:
Sounds good! 👍